charts/agent/templates/_cluster-metrics-config.tpl

```diff
@@ -23,6 +23,109 @@ receivers:
     metrics:
       k8s.node.condition:
         enabled: true
+  {{- if .Values.cluster.metrics.pod.enabled }}
+  prometheus/pod_metrics:
+      config:
+        scrape_configs:
+        - job_name: pod-metrics
+          scrape_interval: {{.Values.cluster.metrics.pod.interval}}
+          honor_labels: true
+          kubernetes_sd_configs:
+          - role: pod
+          relabel_configs:
+          # this is defaulted to keep but if set to drop then pod metrics will not be collected
+          - action: {{.Values.cluster.metrics.pod.action}}
+
+          # Drop anything matching the configured namespace.
+          - action: 'drop'
+            source_labels: ['__meta_kubernetes_namespace']
+            regex: {{.Values.cluster.metrics.pod.namespace_drop_regex}}
+
+          # Drop anything not matching the configured namespace.
+          - action: 'keep'
+            source_labels: ['__meta_kubernetes_namespace']
+            regex: {{.Values.cluster.metrics.pod.namespace_keep_regex}}
+
+          # Drop endpoints without one of: a port name suffixed with the configured regex, or an explicit prometheus port annotation.
+          - action: 'keep'
+            source_labels: ['__meta_kubernetes_pod_container_port_name', '__meta_kubernetes_pod_annotation_prometheus_io_port']
+            regex: '({{.Values.cluster.metrics.pod.port_keep_regex}};|.*;\d+)'
+
+          # Drop pods with phase Succeeded or Failed.
```

Comment:
why do these get dropped? no scraping metrics from job pods basically?

Reply:
This is same as existing - I assume it will just keep scraping same metrics?  I don't actually know.